Program.cs

```diff
@@ -1,10 +1,36 @@
-﻿namespace Training
-{
-    internal class Program
-    {
-        static void Main(string[] args)
-        {
-            Console.WriteLine("Hello, World!");
-        }
-    }
+﻿// Convert decimal to hexadecimal and binary
+//string hexvalue = input.ToString ("X") ----> in-built function for reference
+
+Console.Write ("Enter a decimal value: ");
+if (int.TryParse (Console.ReadLine (), out int input)) {
+   if (input == 0) Console.WriteLine ("Hexadecimal: 0 \nBinary: 0");
+   else {
+      Console.WriteLine ($"Hexadecimal: {hexadecimal ()}");

```

Comment:
You can write this in a single writeline statement for binary and hexadecimal values.